VideoLocker/src/main/java/org/edx/mobile/module/download/DownloadCompleteReceiver.java

```diff
@@ -64,8 +62,6 @@ public void onResult(VideoModel result) {
                                 }
                                 count ++;
                                 p.put(PrefManager.Key.COUNT_OF_VIDEOS_DOWNLOADED, count);
```

Comment:
Hmm, this count doesn't seem be used anywhere, so we could remove this and the key I guess.